content/blog/2023/2023-08-31-fairness-hiding-protected-attribute.md

```diff
@@ -0,0 +1,56 @@
++++
+author = "Žan Mervič"
+date = "2023-08-31"
+draft = false
+title = "Orange Fairness - Hiding Protected Attribute"
+type = "blog"
+thumbImage = "/blog_img/2023/2023-08-31-fairness-hiding-protected-attribute-thumb.png"
+frontPageImage = "/blog_img/2023/2023-08-31-fairness-hiding-protected-attribute-thumb.png"
+blog = ["fairness"]
+shortExcerpt = "In this blog, we confront the common misconception that merely removing a protected attribute from a dataset eliminates bias in model predictions. Our case study reveals that models trained without these attributes still produce biased results. This is due to feature correlations that indirectly capture the protected information. Our conclusion? You cannot sidestep the need for specialized fairness algorithms."
+longExcerpt = "Find out why merely removing protected attributes will not fix bias. Features often correlate, letting models infer biases. Fairness algorithms are key for genuine bias mitigation."
++++
+
+In the previous blog post, we introduced and explained different fairness algorithms that can be used to mitigate bias in a dataset or model predictions. In this blog post, we will discuss a common misconception about fairness: simply removing the protected attribute from the dataset will remove bias. An example will show why this is not the case and why it is essential to use fairness algorithms instead.
```

Comment:
In the previous blog post -> Previously
In this blog post -> Here
common misconception about fairness - > common misconception
simply -> DELETE
An example will show - > We show
instead -> DELETE